lib/metasploit/framework/data_service/proxy/exploit_data_proxy.rb

```diff
@@ -2,7 +2,7 @@ module ExploitDataProxy
 
   def report_exploit_attempt(host, opts)
     begin
-      data_service = self.get_data_service()
+      data_service = self.get_data_service
       data_service.report_exploit_attempt(host, opts)
```

Comment:
Why is there no call to `add_opts_workspace(opts)` in this method?

Reply:
I'm not sure on this one, I'll need to look into it. Standby.

Reply:
I double checked on this and ended up finding a bug. This method is only called when importing an XML output created from Pro. There was a bug earlier on in the calling code that was actually making it exit the import too early and this method was never being called. I'll put a separate PR up to fix that issue.

Once I got around that, I did find that workspace is not needed in the DbManager method. The workspace is inferred from the host object that the exploit_attempt is attached to.